### PR TITLE
Update CHARTER.md

### PR DESCRIPTION
**Pull request by xcorail**
cc @david-a-wheeler @SecurityCRob @blabla1337 @AevaOnline 
My first pass at the PR on the charter, I want to run it by you first for a quick 👍🏾  / 👎🏾  before sharing widely. 
- I saw that some other groups have just adopted the template one, with the reference to a TSC 🤷🏾‍♂️ so for consistency purposes, I tried to keep as close as possible to the template while removing the reference to a TSC that we don't have. This is not what we decided during the meeting so let me know if you prefer that I stick to what we decided in the meeting (scope, points of contact), or if it's good to have some consistency among working groups. 
- Glenn, is the section on IP Policy creating conflicts for SKF?

Thanks

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -8,17 +8,18 @@ This Technical Charter sets forth the responsibilities and procedures for techni
...
+The Best Practices for OSS Developers working group is dedicated to 
+- helping identify and curating an accessible inventory of best practices; 
+- raising awareness; 
+- and education of secure code best practices for open source developers.
 
-- b. The scope of the Technical Initiative includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration, and the creation of other artifacts that support the mission.
+#### 2. Technical initiative roles
 
-#### 2. Technical Steering Committee
+- a. The primary points of contact are:
```
Comment (vmbrasseur) [inline CHARTER.md:18]: Don't enshrine names in the charter, only roles. Making changes to the charter is A Thing. Instead, put the names in the README and/or add a link directing people to the current holders of the listed roles.

File: CHARTER.md [hunk 2]
```diff
@@ -46,21 +43,24 @@ This Technical Charter sets forth the responsibilities and procedures for techni
 
    - v. appointing representatives to work with other open source or open standards communities;
 
-   - vi. approving and implementing policies and processes for contributing (to be published in the Technical Initiative repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
 
-   - vii. facilitating discussions, seeking consensus, and where necessary, voting on technical matters relating to the Technical Initiative; and
+ - f. The working group lead is responsible of
```
Comment (vmbrasseur) [inline CHARTER.md:47]: responsible for

Comment (vmbrasseur) [inline CHARTER.md:47]: There's a section for the responsibilities for the lead, but not for the co-lead. Probably should add that.

File: CHARTER.md [hunk 3]
```diff
@@ -46,21 +43,24 @@ This Technical Charter sets forth the responsibilities and procedures for techni
...
-   - viii. coordinating any communications regarding the Technical Initiative.
+   - ii. coordinating any communications regarding the Technical Initiative.
+  
+   - iii. approving and implementing policies and processes for contributing (to be published in the Technical Initiative repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
 
-#### 3. TSC Voting
+#### 3. Voting
 
-- a. While the Technical Initiative aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Technical Initiative forward, the voting members of the TSC will vote on a one vote per voting member basis.
+- a. While the Technical Initiative aims to operate as a consensus-based community, if any decision requires a vote to move the Technical Initiative forward, the working group will vote on a one vote per member basis.
```
Comment (vmbrasseur) [inline CHARTER.md:57]: To be consistent, most of these instances of "working group" should probably be changed to "Technical Initiative."

File: CHARTER.md [hunk 4]
```diff
@@ -46,21 +43,24 @@ This Technical Charter sets forth the responsibilities and procedures for techni
...
 
-- b. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
+- b. Quorum for working group meetings requires at least fifty percent of the average attendance of the working group to be present.
```
Comment (vmbrasseur) [inline CHARTER.md:59]: This is…way, WAY too fuzzy for comfort. Maybe 50% of Collaborators?

File: CHARTER.md [hunk 5]
```diff
@@ -8,17 +8,18 @@ This Technical Charter sets forth the responsibilities and procedures for techni
...
+  - Working group co-lead: [Xavier René-Corail](https://github.com/xcorail)
 
-- a. The Technical Steering Committee (the "TSC") will be responsible for all oversight of the Technical Initiative. 
-
-- b. The TSC voting members are initially the Technical Initiative’s Maintainers. The Maintainers will be documented in the Technical Initiative repository. The TSC is responsible for determining the future process for defining voting members of the TSC, and any such alternative approach will also be documented appropriately.  Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
-
-- c. The Technical Initiative generally will involve Collaborators and Contributors. The TSC may adopt or modify additional roles so long as the roles are documented in the Technical Initiative’s repository. Unless otherwise documented: 
+- b. The Technical Initiative generally will involve Collaborators and Contributors. The working group may adopt or modify additional roles so long as the roles are documented in the Technical Initiative’s repository. Unless otherwise documented:
```
Comment (vmbrasseur) [inline CHARTER.md:22]: Based on some of the Voting stuff below, as well as the duties listed for them, we should probably create a Team for the Collaborators then use that not only for setting merge permissions on the repo but also as a list for determining voting rights and quorum. If that's the way we decide to go, then we should probably define it in the doc somehow.

File: CHARTER.md [hunk 6]
```diff
@@ -46,21 +43,24 @@ This Technical Charter sets forth the responsibilities and procedures for techni
...
+- a. While the Technical Initiative aims to operate as a consensus-based community, if any decision requires a vote to move the Technical Initiative forward, the working group will vote on a one vote per member basis.
 
-- b. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
+- b. Quorum for working group meetings requires at least fifty percent of the average attendance of the working group to be present. 
 
-- c. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
+- c. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the working group.
```
Comment (vmbrasseur) [inline CHARTER.md:61]: This needs to be more specific. "voting members of the working group" should be "Collaborators," IMO.

**Review by vmbrasseur — CHANGES_REQUESTED** [review]
Such great work, thank you! I've added some comments of things that I feel should be changed and/or clarified. <3

File: CHARTER.md (hunk 1 above)
Comment (xcorail) [inline CHARTER.md:18]:
```suggestion
- a. The primary points of contact are the lead and co-lead of the working group, who are listed in the [README](https://github.com/ossf/wg-best-practices-os-developers/tree/xcorail-patch-1#governance)
```

File: CHARTER.md [hunk 7]
```diff
@@ -8,17 +8,18 @@ This Technical Charter sets forth the responsibilities and procedures for techni
...
+- raising awareness; 
+- and education of secure code best practices for open source developers.
 
-- b. The scope of the Technical Initiative includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration, and the creation of other artifacts that support the mission.
+#### 2. Technical initiative roles
 
-#### 2. Technical Steering Committee
+- a. The primary points of contact are:
+  - Working group lead: [Christopher "CRob" Robinson](https://github.com/RedHatCRob)
+  - Working group co-lead: [Xavier René-Corail](https://github.com/xcorail)
```
Comment (xcorail) [inline CHARTER.md:20]:
```suggestion
```

File: CHARTER.md (hunk 2 above)
Comment (xcorail) [inline CHARTER.md:47]:
```suggestion
 - f. The working group lead is responsible for
```

File: CHARTER.md (hunk 4 above)
Comment (xcorail) [inline CHARTER.md:59]:
```suggestion
- b. Quorum for working group meetings requires at least fifty percent of Collaborators to be present. 
```

File: CHARTER.md (hunk 6 above)
Comment (xcorail) [inline CHARTER.md:61]:
```suggestion
- c. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of Collaborators.
```

**Comment by xcorail** [comment]
Thanks a lot ❤️ @vmbrasseur 
I incorporated your feedback. I'd appreciate another round of 👀 
(also not a native English speaker, so feel free to fix anything)

**Pull request merged**